permission_handler/android/src/main/java/com/baseflow/permissionhandler/PermissionUtils.java

```diff
@@ -176,20 +180,29 @@ static int parseManifestName(String permission) {
                 break;
 
             case PermissionConstants.PERMISSION_GROUP_IGNORE_BATTERY_OPTIMIZATIONS:
-                if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M && hasPermissionInManifest(context, permissionNames, Manifest.permission.REQUEST_IGNORE_BATTERY_OPTIMIZATIONS))
+                if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M && hasPermissionInManifest(context, permissionNames,
+                        Manifest.permission.REQUEST_IGNORE_BATTERY_OPTIMIZATIONS))
                     permissionNames.add(Manifest.permission.REQUEST_IGNORE_BATTERY_OPTIMIZATIONS);
                 break;
 
             case PermissionConstants.PERMISSION_GROUP_ACCESS_MEDIA_LOCATION:
-                if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.Q && hasPermissionInManifest(context, permissionNames, Manifest.permission.ACCESS_MEDIA_LOCATION))
+                if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.Q
+                        && hasPermissionInManifest(context, permissionNames, Manifest.permission.ACCESS_MEDIA_LOCATION))
                     permissionNames.add(Manifest.permission.ACCESS_MEDIA_LOCATION);
                 break;
 
             case PermissionConstants.PERMISSION_GROUP_ACTIVITY_RECOGNITION:
-                if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.Q && hasPermissionInManifest(context, permissionNames, Manifest.permission.ACTIVITY_RECOGNITION))
+                if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.Q
+                        && hasPermissionInManifest(context, permissionNames, Manifest.permission.ACTIVITY_RECOGNITION))
                     permissionNames.add(Manifest.permission.ACTIVITY_RECOGNITION);
                 break;
 
+            case PermissionConstants.PERMISSION_GROUP_SYSTEM_ALERT_WINDOW:
+                if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M
+                        && hasPermissionInManifest(context, permissionNames, Manifest.permission.SYSTEM_ALERT_WINDOW))
+                    permissionNames.add(Manifest.permission.SYSTEM_ALERT_WINDOW);
+                break;
+
```

Comment:
This is the new permission you added which is fine, but make sure it matches formatting with the other case statements